cross_validation.py

```diff
@@ -1,15 +1,17 @@
-import numpy as np
+"""
+Cross validation
+"""
+# import numpy as np
 import torch
-import torch.nn as nn
+# import torch.nn as nn
 import torch.optim as optim
 from tqdm import tqdm
 
-from tst import Transformer
-from tst.loss import OZELoss
-
+from src.benchmark import BiGRU  # , LSTM
 from src.dataset import OzeDataset
-from src.utils import compute_loss, fit, Logger, kfold
-from src.benchmark import LSTM, BiGRU
```

Comment:
This should be made to load the benchmark package, not model individually. But this correction doesn't solve this problem

Reply:
To solve that problem I suggest the creation of a test folder. So the package itself will only have time series transformer related stuff.